content/en/apps/guides/onboarding/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Training App"
+linkTitle: "Using a Training App"
+weight: 15
+description: >
+   Best practices when using a Training App to keep training and production data apart
+
+---
+
+When onboarding new users, having a dedicated CHT app and instance for training can be helpful; it allows new users to do training exercises with mock data to get familiar with the app, while not having the data from their training interfering with the future use of their CHT app. Different approaches are possible, such as only entering real patient data during training, or manually deleting all the training data, but these methods are less practical for large deployments.
+
+{{% alert title="Note" %}}The suggestions in this guide should be assessed and adapted as needed to benefit a deployment. It is important that users don't accidentally use the wrong app. The [troubleshooting guide]() can help to monitor and remediate training data being in the production instance, or the opposite.{{% /alert %}}
+
+### Setting up a training app
+
+A separate Android App can be created for training, which would point to a CHT instance dedicated to training. The training instance should have the same configuration as the production instance, and have users created for training. To differentiate the Android app used for training from the production one, create a duplicate of your brand, and modify the following aspects:
```

Comment:
We should deep link to the technical how to and I think use "flavor" over "brand" to stick with existing vernacular:

```suggestion
A separate Android App can be created for training, which would point to a CHT instance dedicated to training. The training instance should have the same configuration as the production instance, and have users created for training. To differentiate the Android app used for training from the production one, create a duplicate of [your flavor]({{< relref "core/guides/android/branding" >}}) and modify the following aspects
```